### ISSUE TITLE
kops create warns about new Kubernetes version

### ISSUE THREAD

**Issue by olalonde** [kubernetes/kops#2509]
```bash
kops create cluster \
                --cloud "aws" \
                --zones "us-west-2a" \
                --networking "kopeio-vxlan" \
                --vpc "vpc-dd775bba" \
                --node-security-groups "sg-36822b4d" \
                --ssh-public-key=./ssh/olalonde.pub \
                --name "staging.mydomain.com" \
                2> /dev/null || true
Previewing changes that will be made:


*********************************************************************************

A new kubernetes version is available: 1.5.4
Upgrading is recommended (try kops upgrade cluster)

More information: https://github.com/kubernetes/kops/blob/master/permalinks/upgrade_k8s.md#1.5.4

*********************************************************************************

...
```

```
$ kops version
Version 1.5.3 (git-46364f6)
```

**Comment by dolftax** [comment]
I can replicate this.

**Comment by kenden** [comment]
@olalonde could you re-run your command with `-v=10` to get debugg information and post the result?

**Comment by kenden** [comment]
Related https://github.com/kubernetes/kops/issues/2436

**Comment by dolftax** [comment]
```
admin@906bdb5edab4:~$ ./kops update cluster foo --yes -v=10
..
..
..
I0511 10:43:54.642837    4278 options_loader.go:128] executing builder *components.KubeProxyOptionsBuilder
I0511 10:43:54.644675    4278 spec_builder.go:68] options: {
  "channel": "stable",
  "configBase": "s3://foo",
  "cloudProvider": "aws",
  "kubernetesVersion": "1.5.4",
  "subnets": [
    {
      "name": "ap-southeast-1a",
      "zone": "ap-southeast-1a",
      "cidr": "172.20.32.0/19",
      "type": "Public"
    }
  ],
  "masterPublicName": "api.foo",
  "masterInternalName": "api.internal.foo",
  "networkCIDR": "172.20.0.0/16",
  "topology": {
    "masters": "public",
    "nodes": "public",
    "dns": {
      "type": "Public"
    }
  },
  "secretStore": "s3://foo/secrets",
  "keyStore": "s3://foo/pki",
  "configStore": "s3://foo",
  "dnsZone": "XYZ",
  "clusterDNSDomain": "cluster.local",
  "serviceClusterIPRange": "100.64.0.0/13",
  "nonMasqueradeCIDR": "100.64.0.0/10",
  "sshAccess": [
    "0.0.0.0/0"
  ],
  "kubernetesApiAccess": [
    "0.0.0.0/0"
  ],
  "etcdClusters": [
    {
      "name": "main",
      "etcdMembers": [
        {
          "name": "a",
          "instanceGroup": "master-ap-southeast-1a"
        }
      ]
    },
    {
      "name": "events",
      "etcdMembers": [
        {
          "name": "a",
          "instanceGroup": "master-ap-southeast-1a"
        }
      ]
    }
  ],
  "docker": {
    "bridge": "",
    "logLevel": "warn",
    "ipTables": false,
    "ipMasq": false,
    "storage": "overlay,aufs",
    "version": "1.12.3"
  },
  "kubeDNS": {
    "image": "gcr.io/google_containers/kubedns-amd64:1.3",
    "replicas": 2,
    "domain": "cluster.local",
    "serverIP": "100.64.0.10"
  },
  "kubeAPIServer": {
    "image": "gcr.io/google_containers/kube-apiserver:v1.5.4",
    "logLevel": 2,
    "cloudProvider": "aws",
    "securePort": 443,
    "insecurePort": 8080,
    "address": "127.0.0.1",
    "etcdServers": [
      "http://127.0.0.1:4001"
    ],
    "etcdServersOverrides": [
      "/events#http://127.0.0.1:4002"
    ],
    "admissionControl": [
      "NamespaceLifecycle",
      "LimitRanger",
      "ServiceAccount",
      "PersistentVolumeLabel",
      "DefaultStorageClass",
      "ResourceQuota"
    ],
    "serviceClusterIPRange": "100.64.0.0/13",
    "allowPrivileged": true,
    "apiServerCount": 1,
    "anonymousAuth": false,
    "kubeletPreferredAddressTypes": [
      "InternalIP",
      "Hostname",
      "ExternalIP",
      "LegacyHostIP"
    ],
    "storageBackend": "etcd2",
    "authorizationMode": "AlwaysAllow"
  },
  "kubeControllerManager": {
    "master": "127.0.0.1:8080",
    "logLevel": 2,
    "image": "gcr.io/google_containers/kube-controller-manager:v1.5.4",
    "cloudProvider": "aws",
    "clusterName": "foo",
    "clusterCIDR": "100.96.0.0/11",
    "allocateNodeCIDRs": true,
    "configureCloudRoutes": true,
    "leaderElection": {
      "leaderElect": true
    },
    "attachDetachReconcileSyncPeriod": "1m0s"
  },
  "kubeScheduler": {
    "master": "http://127.0.0.1:8080",
    "logLevel": 2,
    "image": "gcr.io/google_containers/kube-scheduler:v1.5.4",
    "leaderElection": {
      "leaderElect": true
    }
  },
  "kubeProxy": {
    "image": "gcr.io/google_containers/kube-proxy:v1.5.4",
    "cpuRequest": "100m",
    "logLevel": 2,
    "clusterCIDR": "100.96.0.0/11"
  },
  "kubelet": {
    "apiServers": "https://api.internal.foo",
    "logLevel": 2,
    "podManifestPath": "/etc/kubernetes/manifests",
    "hostnameOverride": "@aws",
    "allowPrivileged": true,
    "enableDebuggingHandlers": true,
    "clusterDomain": "cluster.local",
    "clusterDNS": "100.64.0.10",
    "networkPluginName": "kubenet",
    "cloudProvider": "aws",
    "cgroupRoot": "docker",
    "babysitDaemons": true,
    "nonMasqueradeCIDR": "100.64.0.0/10",
    "networkPluginMTU": 9001,
    "evictionHard": "memory.available\u003c100Mi,nodefs.available\u003c10%,nodefs.inodesFree\u003c5%,imagefs.available\u003c10%,imagefs.inodesFree\u003c5%"
  },
  "masterKubelet": {
    "apiServers": "http://127.0.0.1:8080",
    "logLevel": 2,
    "podManifestPath": "/etc/kubernetes/manifests",
    "hostnameOverride": "@aws",
    "allowPrivileged": true,
    "enableDebuggingHandlers": true,
    "clusterDomain": "cluster.local",
    "clusterDNS": "100.64.0.10",
    "networkPluginName": "kubenet",
    "cloudProvider": "aws",
    "cgroupRoot": "docker",
    "babysitDaemons": true,
    "registerSchedulable": false,
    "nonMasqueradeCIDR": "100.64.0.0/10",
    "networkPluginMTU": 9001,
    "evictionHard": "memory.available\u003c100Mi,nodefs.available\u003c10%,nodefs.inodesFree\u003c5%,imagefs.available\u003c10%,imagefs.inodesFree\u003c5%"
  },
  "networking": {
    "kubenet": {}
  },
  "api": {
    "dns": {}
  },
  "authorization": {
    "alwaysAllow": {}
  }
}
I0511 10:43:54.646055    4278 channel.go:154] RecommendedVersion="1.5.1", Have="1.5.0".  Recommending upgrade
I0511 10:43:54.646108    4278 channel.go:185] VersionRecommendationSpec does not specify RequiredVersion

*********************************************************************************

A new kops version is available: 1.5.1

Upgrading is recommended
More information: https://github.com/kubernetes/kops/blob/master/permalinks/upgrade_kops.md#1.5.1

*********************************************************************************

I0511 10:43:54.646202    4278 channel.go:137] RecommendedVersion="1.5.4", Have="1.5.4".  No upgrade needed.
I0511 10:43:54.646232    4278 channel.go:177] RequiredVersion="1.5.1", Have="1.5.4".  No upgrade needed.
..
..
..

Cluster changes have been applied to the cloud.


Changes may require instances to restart: kops rolling-update cluster
```
